lib/ABI/FunctionType.cpp

```diff
@@ -171,6 +148,7 @@ class ConversionHelper {
   using DistributedArguments = llvm::SmallVector<DistributedArgument, 4>;
 
   using ArgumentVector = SortedVector<model::Argument>;
+  using TypeVector = llvm::SmallVector<UpcastablePointer<model::Type>, 16>;
 
 public:
   static std::optional<model::TypePath>
```

Comment:
NOTETOSELF: review in its entirety `toCABI`.

Reply:
Should this be closed? 🤔